### PR TITLE
Added various missing ADRs

### PR DESCRIPTION
**Pull request by joates-madetech**
# Description

The ADRs coving a range of topics up to Event Streaming and initial Extraction of the Tenure Data.

### CODE REVIEW

File: docs/architecture-decisions/records/012-do-not-connect-to-production-databases.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Decision
+
+We have reached the conclusion that we should not be connecting to production databases to extract data. The major
+reason for this is that connecting to a production database will increase the overall load that the database is under
+which could cause the related service to fail which would affect residents of Hackney negatively.
+
+## Consequences
+
+- Getting access to data is considerable harder unless the service implements Event Streaming as we have a limited set of options for ingesting data without a connection to the production system.
```
Comment (elcorbs) [inline docs/architecture-decisions/records/012-do-not-connect-to-production-databases.md:36]:
```suggestion
- Getting access to data is considerably harder unless the service implements event streaming as we have a limited set of options for ingesting data without a connection to the production system.
```

File: docs/architecture-decisions/records/012-move-staging-to-pre-production.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+description: ""
+tags: [adr]
+number: "12"
+date-issued: "2021-12-16"
+status: "Accepted"
+---
+
+## Context
+
+Towards the end of December 2021 a component of our infrastructure the provided connectivity between a production and
```
Comment (elcorbs) [inline docs/architecture-decisions/records/012-move-staging-to-pre-production.md:13]:
```suggestion
Towards the end of December 2021 a component of our infrastructure that provided connectivity between a production and
```

File: docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+The Data Platform team are interested in ingesting data from a variate of Hackney APIs. We will need to access this data
+without connecting directly to the production databases see [ADR 012](/architecture-decisions/records/do-not-connect-to-production-data)
+
+We have therefore explored different options for transferring the data into the Data Platform.
+
+## Decision Drivers
+
+- Security
+-
```
Comment (elcorbs) [inline docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md:21]:
```suggestion

```

File: docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+## Context
+
+The Data Platform team are interested in ingesting data from a variate of Hackney APIs. We will need to access this data
+without connecting directly to the production databases see [ADR 012](/architecture-decisions/records/do-not-connect-to-production-data)
+
+We have therefore explored different options for transferring the data into the Data Platform.
+
+## Decision Drivers
+
+- Security
```
Comment (elcorbs) [inline docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md:20]: Are there not more drivers than just security for this?

File: docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- 
+
+## Considered Options
+
+- Access data via read-only replicas
+- Scrap the APIs to get the state of each object
```
Comment (elcorbs) [inline docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md:26]:
```suggestion
- Scrape the APIs to get the state of each object
```

File: docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+---
+id: using-event-streaming-to-ingest-data
+title: "Using Event Streaming to Ingest Data"
+description: ""
+tags: [adr]
+number: "13"
+date-issued: "2022-02-??"
```
Comment (elcorbs) [inline docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md:7]: Pick a date here?

File: docs/architecture-decisions/records/015-initial-extract-of-tenure-data.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+---
+id: initial-extract-of-tenure-data
+title: "Initial Extract of Tenure Data"
+description: ""
+tags: [adr]
+number: "14"
+date-issued: "2022-02-??"
```
Comment (elcorbs) [inline docs/architecture-decisions/records/015-initial-extract-of-tenure-data.md:7]: Pick a date here?

File: docs/architecture-decisions/records/015-initial-extract-of-tenure-data.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+## Consequences
+
+### Short Term
+- The Data Platform will need direct access to the APIs data storage in order to perform the ingestion
+- The process is not designed to be maintainable and should not be repeated multiple times
+- Any data inconsistencies will be difficult to resolve.
+
+### Long Term
+- All APIs will need to be designed to emmit current state events when required.
```
Comment (elcorbs) [inline docs/architecture-decisions/records/015-initial-extract-of-tenure-data.md:52]:
```suggestion
- All APIs will need to be designed to emit current state events when required.
```

**Review by elcorbs — APPROVED** [review]

File: docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md (hunk 6 above)
Comment (joates-madetech) [reply docs/architecture-decisions/records/014-using-event-streaming-to-ingest-data.md:7]:
```suggestion
date-issued: "2022-01-01"
```

**Review by elcorbs — APPROVED** [review]

**Pull request merged**